zgnroot/include/zgnr/gl-base-technique.h

```diff
@@ -8,6 +8,18 @@
 extern "C" {
 #endif
 
+enum zgnr_gl_base_technique_draw_method {
+  ZGNR_GL_BASE_TECHNIQUE_DRAW_ARRAYS,
+};
```

Comment:
```suggestion
enum zgnr_gl_base_technique_draw_method {
  ZGNR_GL_BASE_TECHNIQUE_DRAW_METHOD_NONE,
  ZGNR_GL_BASE_TECHNIQUE_DRAW_METHOD_ARRAYS,
};
```

We need an initial state of `zgnr_gl_base_technique::current.draw_method`, which means no drawing method specified yet.